src/hardware/envelope.cpp

```diff
@@ -100,7 +100,7 @@ void Envelope::Apply(const bool is_stereo, AudioFrame &frame)
 	if (++frames_done > expire_after_frames || edge >= edge_limit) {
 		process = &Envelope::Skip;
 		(void)channel_name; // [[maybe_unused]] in release builds
-		DEBUG_LOG_MSG("ENVELOPE: %s done after %u frames, peak sample was %f",
+		DEBUG_LOG_MSG("ENVELOPE: %s done after %u frames, peak sample was %.1f",
```

Comment:
That's too narrow; I'd use 2 decimal places at least, but maybe go for 4?

Reply:
These are the scaled up 16-bit-range samples (the whole integers are still being shown) - For example, "peak sample was 28421.3" (Aces of Deep intro w/ MIDI).

It's only in debugging builds, and I've personally only used this to see if the game tried to push beyond 16-bit or to get a rough gauge for initial volume. 

But with this info - do you still want more (if so - happy to add!)

Reply:
Generally, logging audio level related info is best to be done as decibels with 2 decimal places, as the human perception of volume is logarithmic.

Reply:
bumped it to `%.4f` in the last version before I pushed to main.

>  logging audio level related info is best to be done as decibels with 2 decimal places, as the human perception of volume is logarithmic

Thanks @johnnovak - the more we can fall in line with industry standards, the easier it will be for users to parse.